org.kde.kdiamond.json

```diff
@@ -10,29 +10,28 @@
         "--socket=x11",
         "--socket=wayland",
         "--socket=pulseaudio",
-        "--device=dri",
-        "--filesystem=xdg-config/kdeglobals:ro"
```

Comment:
Why did you remove this?

Reply:
As far as I know, it's set by default now in 5.12. Please correct me if I'm wrong though.

Reply:
flatpak-builder will inherit some properties from the runtime, such as kdeglobals. Nevertheless, `--device=dri` isn't one of these so you probably still want it if you want graphics hardware acceleration.

Reply:
`--device=dri` is still there, but I had to remove the comma to not bork the json.